docs/_sass/_main.scss

```diff
@@ -481,4 +503,38 @@ textarea {
         margin-left: auto;
         margin-right: auto;
     }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 800px) {
+    #floating-concierge-button-lhn {
+        display: none;
+    }
+
+    .floating-concierge-button {
+        position: fixed;
+    }
+}
+
+@media screen and (min-width: 801px) {
```

Comment:
Would love to see mixins for these breakpoints as well, instead of all these hardcoded instances of 800px and 801px 😬

Reply:
Okay, that took me _way_ too long, but it's done now!